compiler/rustc_trait_selection/src/solve/assembly.rs

```diff
@@ -76,7 +76,7 @@ pub(super) enum CandidateSource {
     ///     let _y = x.clone();
     /// }
     /// ```
-    AliasBound(usize),
+    AliasBound(),
```

Comment:
Rather than a tuple variant with no fields, it'd be more idiomatic for this to be a plain field-less variant (ie omit the brackets).